src/parser.rs

```diff
@@ -661,6 +662,12 @@ impl<'a> DocParser<'a> {
 
     let mut is_ambient = true;
 
+    if let Some((js_doc, span)) = leading_js_doc(parsed_source) {
+      let doc_node =
+        DocNode::module_doc(get_location(parsed_source, span.lo), js_doc);
+      doc_entries.push(doc_node);
+    }
```

Comment:
I was confused at first when I saw it. Maybe `leading_js_doc` could use a better name suggesting that it extracts "module" tag?

Reply:
I've renamed it and also provided a doc block to clarify.